plugins/zip-extractor/src/zip.extractor.e2e.spec.ts

```diff
@@ -40,6 +40,6 @@ describe('ZipExtractor e2e', () => {
       await listener.waitFor('file:created', 4)
       const filesPostUpload = await getFiles(spaceId)
       expect(filesPostUpload.length).toBe(4)
-    })
+    }, 15_000)
```

Comment:
15s is a while! 😭

Reply:
Usually takes the default 5, but sometimes it takes a little longer. Just got tired of the test failing occasionally.